components/issue-list-item.tsx

```diff
@@ -34,7 +36,7 @@ export default function IssueListItem({issue = null}:{issue?: IssueData}) {
             .catch((err)  => console.log('err ->', err))
     }
      getIssueBeprosStaked(issue?.issueId)
-   },[issue])
+   },[metaMaskWallet])
```

Comment:
shouldn't users be able to see the issue even if metamask isn't loaded?

Reply:
the requests for the methods improve the link criteria to work